trie.go

```diff
@@ -124,6 +124,13 @@ func (p *prefixTrie) CoveredNetworks(network net.IPNet) ([]RangerEntry, error) {
 	return p.coveredNetworks(net)
 }
 
+// Covering returns the list of RangerEntry(s) the given ipnet
+// is covered by. It's like ContainingNetworks() for ipnet.
+func (p *prefixTrie) CoveringNetworks(network net.IPNet) ([]RangerEntry, error) {
+	net := rnet.NewNetwork(network)
+	return p.coveringNetworks(net)
```

Comment:
Is new implementation needed, could we not just do the following?
```suggestion
        return p.ContainingNetworks(network.IP.Mask(network.IPMask))
```

Reply:
I thought about this method. But i've discover [Covers](https://github.com/yl2chen/cidranger/blob/d1cb2c52f37ac257db665ddfaf4af22e89863471/net/ip.go#L213) function, and decided to use it.